tests/subscriptions/test_data.py

```diff
@@ -124,7 +124,7 @@
                         ),
                     ],
                 ),
-                time_window_secs=3600,
+                time_window_secs=10800,
```

Comment:
The `BaseSubscriptionTest` class that the test inherits from adds events to the previous hour. This test worked before because we were rounding timestamp to the nearest granularity. 

The larger time window will catch the events I expect to catch.